### PR TITLE
chore(*): update to react 16.2

### PR DESCRIPTION
**Pull request by christopherthielen**
I fixed the react 16 issues in `@uirouter/react-hybrid` and released `0.1.0`

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -7302,6 +7316,13 @@ react@^0.14.8:
     object-assign "^4.1.1"
     prop-types "^15.6.0"
 
+react@^0.14.8:
```
Comment (jrsquared) [inline yarn.lock]: what snuck this in here?!

Comment (christopherthielen) [reply yarn.lock]: `n3-charts@2.0.28`

File: .travis.yml [hunk 2]
```diff
@@ -6,6 +6,8 @@ dist: trusty
 before_install:
 - export CHROME_BIN=chromium-browser
 - export DISPLAY=:99.0
+- export PATH=$HOME/.yarn/bin:$PATH
+- curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.5.1
```
Comment (christopherthielen) [inline .travis.yml]: force yarn 1.5.1 (or latest?)

File: package.json [hunk 3]
```diff
@@ -13,7 +13,7 @@
   "engines": {
     "node": ">=8.9.0",
     "npm": ">=5.5.1",
-    "yarn": ">=1.3.2"
+    "yarn": ">=1.5.0"
```
Comment (christopherthielen) [inline package.json]: yarn 1.3.2 wasn't hoisting transitive dependencies to the top level,which caused unit tests to fail

**Comment by anotherchrisberry** [comment]
I bet this is fine

**Comment by christopherthielen** [comment]
no u are

**Pull request merged**